hudi-common/src/main/java/org/apache/hudi/avro/HoodieAvroUtils.java

```diff
@@ -164,6 +164,18 @@ public static GenericRecord jsonBytesToAvro(byte[] bytes, Schema schema) throws
     return reader.read(null, jsonDecoder);
   }
 
+  /**
+   * True if the schema contains this name of field
+   */
+  public static boolean containsFieldInSchema(Schema schema, String fieldName) {
+    try {
+      Field field = schema.getField(fieldName);
+      return field != null;
+    } catch (Exception e) {
+      return false;
+    }
```

Comment:
What is the philosophy to return `false` when exception throws ? And do we need such a method for this simple calling ?

Reply:
Since avro 1.10, arvo will throw AvroRuntimeException rather than return null like the previous version if record doesn't contain this key.

Reply:
Do we have different avro version for current code path ?

Reply:
Which avro version will be used depends on the build environment, not the codes !
see the pom and search `avro.version`, you will see when will use the different avro version.

Reply:
I don't think this is solid code, at least you should catch specific exception instead of `Exception`.